lib/t/accounts/profile.ex

```diff
@@ -5,27 +5,30 @@ defmodule T.Accounts.Profile do
   @primary_key false
   @foreign_key_type Ecto.Bigflake.UUID
   schema "profiles" do
-    belongs_to :user, T.Accounts.User, primary_key: true
+    belongs_to(:user, T.Accounts.User, primary_key: true)
 
-    field :story, {:array, :map}
-    field :location, Geo.PostGIS.Geometry
+    field(:story, {:array, :map})
+    field(:location, Geo.PostGIS.Geometry)
 
     # TODO remove
     embeds_one :filters, Filters, primary_key: false, on_replace: :delete do
       # ["F"] or ["F", "M"], etc.
-      field :genders, {:array, :string}
+      field(:genders, {:array, :string})
```

Comment:
These need to be casted in `essential_info_changeset` in `cast_embed(:filters,`, although I'd remove the embed, it was a quick workaround when adding genders, it's not really used anymore.

Reply:
@ruslandoga so maybe use :virtual instead?

Reply:
or maybe just remove `filters` column from `profiles` table altogether?

Reply:
2